tools/mo/openvino/tools/mo/ops/pooling.py

```diff
@@ -91,14 +91,18 @@ def backend_attrs(self):
             ('exclude-pad', lambda node: bool_to_str(node, 'exclude_pad')),
 
             'rounding_type',
-            ('auto_pad', lambda node: node.auto_pad if node.has_valid('auto_pad') else 'explicit'),
-
-            ('dilations', lambda node: ','.join(map(str, node['dilation'][node.spatial_dims]))),
-            'axis',
-
-            ('index_element_type', lambda node: np_data_type_to_destination_type(node.index_element_type))
+            ('auto_pad', lambda node: node.auto_pad if node.has_valid('auto_pad') else 'explicit')
         ]
 
+        if self.attrs.get('pool_method') == 'avg':
+            return backend_attrs_list
+        else:
+            return backend_attrs_list + [
+                ('dilations', lambda node: ','.join(map(str, node['dilation'][node.spatial_dims]))),
+                'axis',
+                ('index_element_type', lambda node: np_data_type_to_destination_type(node.index_element_type))
```

Comment:
There is missing `('auto_pad', lambda node: node.auto_pad if node.has_valid('auto_pad') else 'explicit')`
Is it expected?